lib-multisrc/vercomics/src/eu/kanade/tachiyomi/multisrc/vercomics/VerComics.kt

```diff
@@ -8,45 +8,75 @@ import eu.kanade.tachiyomi.source.model.SChapter
 import eu.kanade.tachiyomi.source.model.SManga
 import eu.kanade.tachiyomi.source.model.UpdateStrategy
 import eu.kanade.tachiyomi.source.online.ParsedHttpSource
-import okhttp3.Headers
 import okhttp3.HttpUrl.Companion.toHttpUrl
 import okhttp3.Request
 import org.jsoup.nodes.Document
 import org.jsoup.nodes.Element
 import rx.Observable
 
-open class VCPVMP(override val name: String, override val baseUrl: String) : ParsedHttpSource() {
-
-    override val lang = "es"
+abstract class VerComics(
+    override val name: String,
+    override val baseUrl: String,
+    override val lang: String,
+) : ParsedHttpSource() {
 
     override val supportsLatest: Boolean = false
 
-    override fun headersBuilder(): Headers.Builder {
-        return Headers.Builder()
-            .add("Referer", "$baseUrl/")
-    }
-
-    override fun latestUpdatesRequest(page: Int) = throw UnsupportedOperationException()
-
-    override fun latestUpdatesSelector() = throw UnsupportedOperationException()
-
-    override fun latestUpdatesFromElement(element: Element) = throw UnsupportedOperationException()
+    protected open val urlSuffix = ""
+    protected open val genreSuffix = ""
 
-    override fun latestUpdatesNextPageSelector() = throw UnsupportedOperationException()
+    override fun headersBuilder() = super.headersBuilder()
+        .add("Referer", "$baseUrl/")
 
     override fun popularMangaRequest(page: Int) = GET("$baseUrl/$urlSuffix/page/$page", headers)
 
     override fun popularMangaSelector() = "div.blog-list-items > div.entry"
```

Comment:
You can use this selector for all sites:
```css
header:has(>h1):has(~ center > div[role=navigation]) + div > div.entry
```

Or the least strict:
```css
header:has(h1) ~ * .entry
```

or somewhere in-between